packages/protocol/contracts/rrp/ConvenienceUtils.sol

```diff
@@ -6,48 +6,62 @@ import "./authorizers/interfaces/IRrpAuthorizer.sol";
 
 /// @title Contract that implements convenience functions
 contract ConvenienceUtils is IConvenienceUtils {
-    struct Announcement {
-        string xpub;
-        address[] authorizers;
-    }
-
-    mapping(address => Announcement) private airnodeToAnnouncement;
+    mapping(address => string) private airnodeToPublicKey;
+    mapping(address => address[]) private airnodeToAuthorizers;
 
-    /// @notice Called by the Airnode operator to set announcement
+    /// @notice Called by the Airnode operator to set it's public key
     /// @dev It is expected for the Airnode operator to call this function with
     /// the respective Airnode's default BIP 44 wallet (m/44'/60'/0'/0/0).
-    /// This announcement does not need to be made for the protocol to be used,
+    /// This public key set does not need to be made for the protocol to be used,
     /// it is mainly for convenience.
     /// @param xpub Extended public key of the Airnode
+    function setAirnodePublicKey(string calldata xpub) external override {
+        airnodeToPublicKey[msg.sender] = xpub;
+        emit SetAirnodePublicKey(msg.sender, xpub);
+    }
+
+    /// @notice Called by the Airnode operator to set authorizers
```

Comment:
So we can probably mention it here that these are announcements and not used for actual authorization